tests/TestCustomChar.py

```
@@ -117,16 +117,17 @@ def test_manager_crud(self):
         # 模擬截圖特徵值的加入
-        fake_mat = np.zeros((10, 10, 3), dtype=np.uint8)
-        fid = self.manager.add_feature_to_character("char1", fake_mat, 1920, 1080)
+        np.random.seed(42)
+        fake_mat = np.random.randint(0, 256, (10, 10, 3), dtype=np.uint8)
+        fid = self.manager.add_feature_to_character("char1", fake_mat, self.task.width, self.task.height)
         char_info_features = self.manager.get_character_info("char1")
         assert char_info_features is not None
         self.assertIn(fid, char_info_features["feature_ids"])
 
         # 測試特徵匹配邏輯 match_feature
         # 目前特徵庫內有一張假圖，如果餵入一模一樣的黑圖，應該回報 True
```

Ruff (0.15.12)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)